lib/sycamore/sycamore/tests/manual/pdf2image.py

```diff
@@ -0,0 +1,41 @@
+import sys
```

Comment:
document why a manual test and not an integration test. Or switch to an integration test.